redhat-developer/odo
# Refactor completion

What is the purpose of this change? What does it change?
Improve and simplify completion handler management, extracting it to a specific package. This should make it easier to work on #871.

Was the change discussed in an issue?
In #871 

How to test changes?
`make test` and check that completions for `service create` and `service delete` still work as expected.
